.gitignore

```diff
@@ -61,5 +61,9 @@ chaindata/
 genesis.json
 keystore/
 
+# Test remnants
+logs/
+migrations/
```

Comment:
Are you sure? That can't be. This `.gitignore` rule is not that generic it will only match direct descendants. In order to achieve what you say you would need: `**/migrations/` as the the [gitignore manual](https://git-scm.com/docs/gitignore).

I also just tried it and any changes under `./populus/migrations` are seen by git.